ginga/AstroImage.py

```diff
@@ -38,8 +38,7 @@ def set_ioClass(cls, klass):
         cls.ioClass = klass
 
     def __init__(self, data_np=None, metadata=None, logger=None,
-                 name=None, wcsclass=None, ioclass=None,
-                 inherit_primary_header=False, save_primary_header=True):
```

Comment:
I don't see `stginga` explicitly calling this in the constructor. Looks like `stginga` only sets it via `general.cfg` and you did not change the config name there, so that's good.

However, there is no guarantee that all your downstream packages/users also only set this via `general.cfg`. This is a public API change without a deprecation period. I am not sure if it is a good practice.

Reply:
Recommendation: Instead of removing the keywords, deprecate them but preserve a backward-compatible functionality with deprecation warning, change log, and API documentation informing users and downstream devs. Wait for maybe two releases (your call on exactly how many) before removing them altogether.

Pro: No surprise for anyone.
Con: More work for you.